docs/dashboards/index.md

```diff
@@ -1,7 +1,9 @@
 title: Sematext Dashboards
 description: Sematext Cloud is a modern monitoring, log management, transaction tracing, real user monitoring, and synthetic monitoring system. Dashboards are a powerful feature for cohesive data analysis from multiple data sources such as logs, metrics, and events.
 
```

Comment:
On this page, should we explain the difference between top-level daashboards and dashboards that we call reports?

Reply:
Something like this?
Note that top level dashboards differ from what we call [App](https://sematext.com/docs/guide/app-guide/) specific dashboards/reports. App specific reports are placed within each app, displaying data specific to that particular App. In contrast, dashboards can fetch data from multiple different Apps.

Reply:
Something like that yes. app ==> App.